lib/game.rb

```diff
@@ -12,16 +12,43 @@ class Game
               :venue_link
 
   def initialize(details)
-    @game_id = details[:game_id].to_i
-    @season = details[:season].to_i
+    @game_id = details[:game_id]
+    @season = details[:season]
     @type = details[:type]
     @date_time = details[:date_time]
-    @away_team_id = details[:away_team_id].to_i
-    @home_team_id = details[:home_team_id].to_i
-    @away_goals = details[:away_goals].to_i
-    @home_goals = details[:home_goals].to_i
+    @away_team_id = details[:away_team_id]
+    @home_team_id = details[:home_team_id]
+    @away_goals = details[:away_goals]
+    @home_goals = details[:home_goals]
     @venue = details[:venue]
     @venue_link = details[:venue_link]
+    @outcome = outcome
+  end
+
+  def outcome
```

Comment:
Clever to add methods in the game class itself so that they could be called on any instance of a game object! A great refactoring idea for the rest of us to implement!